src/lib/functions/registry.ts

```diff
@@ -297,8 +297,8 @@ export class FunctionsRegistry {
   async getFunctionForURLPath(url, method) {
```

Comment:
The parameter should probably be named to `urlPath`, as it will now break if it's a fully qualified URL? Alternatively, we can keep it as `url` and check whether if it's a full URL or a URL path, adjusting accordingly.

Reply:
done in https://github.com/netlify/cli/pull/6212/commits/f2056907cc0e01f51127e6c779735b7916a0df97